### ISSUE TITLE
getChats Function do not return value via tdJson.so

### ISSUE THREAD

**Issue by blacksailer** [tdlib/td#14]
Hello!
I have a problem. I built a library and linked to libtdjson.so. (QtCreator, gcc 4.9).  
I successfully called setTdlibParameters, setDatabaseEncryptionKey, setAuthenticationPhoneNumber, checkAuthenticationCode functions and connected to Telegram network.

I'm trying to get a list of chats via getChats to get list of ids and expect to get results via td_json_client_receive(). But I see output only via inner log and resulted request do not returned through td_json_client_receive(). I see desired output generated by this [line](https://github.com/tdlib/td/blob/master/td/telegram/Td.cpp#L4259).

What am I doing wrong?  Or td_json_client_receive method is designed to work only with UpdateObjects?

**Comment by levlam** [comment]
It is supposed to work.

Could you check that the cycle, from which td_json_client_receive is called, is not blocked, and td_json_client_receive is really called again after some updates/responses received or timeout expired?

**Comment by blacksailer** [comment]
I run function with QtConcurrent::run()  (This method run function with td_json_client_receive() always in separate thread) with QTimer with interval 100. So this function is always called and not blocked.

I can get objects with @type :[ updateUser, updateOption,updateNotificationSettings ].
in Debug I also see response with @type : [ updateChatLastMessage, updateChatReadInbox ] but it is not returned via td_json_client_receive().

I added VLOG [here](https://github.com/tdlib/td/blob/master/td/telegram/ClientJson.cpp#L44) in hope to see unreturning responses via td_json_client_receive(), but with no luck.

To be fully honest, I am trying to build on SailfishOS but this OS has default toolchain with gcc 4.8. So I built library with custom gcc 4.9 and add static libstdc++ and libgcc in cmake so I can trying to use it in SailfishSDK (IDE to build native apps).
